pandas/core/arrays/categorical.py

```diff
@@ -657,11 +657,27 @@ def from_codes(cls, codes, categories, ordered=False):
             categories or -1 for NaN
         categories : index-like
             The categories for the categorical. Items need to be unique.
-        ordered : boolean, (default False)
+        ordered : boolean, optional
             Whether or not this categorical is treated as a ordered
```

Comment:
small nitpick since you're already editing stuff in this section: "a ordered" --> "an ordered"